includes/class-activitypub.php

```diff
@@ -555,15 +555,12 @@ private static function register_post_types() {
 	}
 
 	/**
-	 * Add the 'activitypub' capability to users who can publish posts.
+	 * Add the 'activitypub' capability to new users.
 	 *
 	 * @param int $user_id User ID.
 	 */
 	public static function user_register( $user_id ) {
-		if ( \user_can( $user_id, 'publish_posts' ) ) {
-			$user = \get_user_by( 'id', $user_id );
-			$user->add_cap( 'activitypub' );
-		}
+		\get_user_by( 'id', $user_id )->add_cap( 'activitypub' );
```

Comment:
By default we only want to add the "activitypub" capability to users that can edit posts. The admin can change this afterwards.

Reply:
Oh, I misunderstood your comment then